services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/utils.py

```diff
@@ -112,7 +112,9 @@ async def docker_client() -> AsyncGenerator[aiodocker.Docker, None]:
         await docker.close()
 
 
-async def async_command(command: str, command_timeout: float) -> Tuple[bool, str]:
+async def async_command(
+    command: str, command_timeout: Optional[float]
+) -> Tuple[bool, str]:
```

Comment:
maybe use a NamedTuple or TypedDict or Dataclass in order to be more explicit here

Reply:
will use namedtuple